src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.searchForm {
+  position: relative;
+  margin-top: 20px;
+  height: 70px;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.img {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  right: 17px;
+  height: 19px;
+  width: 19px;
+}
+
+.input {
+  border-radius: 4px;
+  width: 100%;
+  font-family: inherit;
+  height: inherit;
+  font-size: 16px;
+  font-weight: 900;
+  line-height: 21.86px;
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  border: 1px solid #e1e7ed;
+  outline: none;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  color: #3d4e61;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input::placeholder {
+  font-family: inherit;
+  color: #3d4e61;
+  font-weight: 400;
+}
+
+.searchForm--small {
+  height: 42px;
+}
+
+.input--small {
+  font-family: inherit;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+  padding-top: 11px;
+  padding-bottom: 11px;
```

Comment:
you can also combine into `padding` property

Reply:
thanks!